

class AuthenticatorException(Exception):
    pass


class AuthRefused(AuthenticatorException):
    pass
